src/Entity/EntityId.php

```diff
@@ -68,7 +80,11 @@ public function getSerialization() {
 	public static function splitSerialization( $serialization ) {
 		self::assertValidSerialization( $serialization );
 
-		$parts = explode( ':', self::normalizeIdSerialization( $serialization ) );
+		return self::getSerializationParts( self::normalizeIdSerialization( $serialization ) );
+	}
+
```

Comment:
Please document at least the return type.

Reply:
done